### DISCUSSION TITLE
Support URI fragments (scroll down to resource)

### DISCUSSION THREAD

**Discussion by PhilC813** [jocmp/capyreader#615] [Feature Requests]
[URI fragments](https://w.wiki/B67d) are used to specify a specific resource (section) within a given webpage. They're often used in the index at the beginning of listicles, tutorials or similar pages in order to easy make it down to the section you are interested in.

An example (which I used for an issue I opened earlier):
https://www.androidauthority.com/new-android-apps-658839/#4

After the webpage is loaded by the web browser, it'll scroll down to the resource indicated in the fragment, which is the fourth section in this case.

----------

I'm not sure if that's even remotely possible with the current article viewer, but if it can, implementation this behaviour in Capy would be pretty cool!

**Comment by jocmp** [comment]
hey @PhilC813 - this would be cool. It might be possible given that the reader view is also a web page. I'd need to look into it, but I think the biggest challenge would be deciding if a fragment is _within_ the current article.

I added a issue https://github.com/jocmp/capyreader/issues/617 to investigate. I'm all for support better navigation if it's possible.
